configure.ac

```diff
@@ -72,7 +72,7 @@ AC_ARG_ENABLE([perfflow],
     [enable_perfflow=no]
 )
 # TODO Add support for libb64 back once base64 encoding/decoding is fully complete
-# AC_ARG_VAR([LIBB64_DIR], [root directory for libb64])
+AC_ARG_VAR([LIBB64_DIR], [root directory for libb64])
```

Comment:
@JaeseungYeom it would probably be better to use `AC_SEARCH_LIBS` and `AC_CHECK_HEADERS` to locate libb64. Those macros seem to be the recommended way to locate external libraries if they don't provide a `.pc` file. This webpage seems pretty useful in explaining how to use those macros: https://autotools.info/autoconf/finding.html

Reply:
Made changes to allow the use of system-wide libb64, which is probably not common given the library packaging.